src/plugins/analysis/cve_lookup/test/test_setup_repository.py

```diff
@@ -1047,7 +1047,7 @@
 ]
 
 
-@pytest.fixture(scope='session', autouse=True)
```

Comment:
If you don't want to explicit you can omit `scope=function`. If you want to be explicit a comment might be useful to explain why it is important to have this be function scoped.

Reply:
removed